### PR TITLE
Added Content Groups for iOS and Android

### PR DESCRIPTION
**Pull request by nsantacruz**
I believe this works in setting the content group on the tracker but I couldn't find Google Analytics documentation on how to this information.

### CODE REVIEW

File: android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java [hunk 1]
```diff
@@ -251,6 +251,14 @@ public void trackSocialInteraction(String trackerId, String network, String acti
         }
     }
 
+	@ReactMethod
+	public void trackContentGroup(String trackerId, int contentGroupIndex, String category) {
+		Tracker tracker = getTracker(trackerId);
+		if (tracker != null) {
+			tracker.set(contentGroup(contentGroupIndex),category);
```
Comment (cbrevik) [inline android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java]: Have you tested this for Android? Where does `contentGroup` come from?

Also, could you fix whitespaces for this methods, `trackContentGroup` is indented further than the methods around.

File: src/GoogleAnalyticsTracker.js [hunk 2]
```diff
@@ -66,6 +66,15 @@ export class GoogleAnalyticsTracker {
     GoogleAnalyticsBridge.trackEvent(this.id, category, action, optionalValues);
   }
 
+	/**
```
Comment (cbrevik) [inline src/GoogleAnalyticsTracker.js]: Whitespaces here also, pushes the function further in than the ones around,.

File: android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java (hunk 1 above)
Comment (nsantacruz) [inline android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java]: Like you said, documentation about this is really scarce. Here's the page I got most of the info from. You can see in the Android snippet it uses `contentGroup`. Not sure if this is global or not. https://support.google.com/analytics/answer/2853546?hl=en

**Comment by cbrevik** [comment]
Hi, no activity here for a while so I'm closing this.

**Pull request closed without merging**